src/charts/iot-edge-connector/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.rbac.install }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "edge-provider.fullname" . }}-role-binding
+subjects:
+- kind: ServiceAccount
+  name: {{ template "edge-provider.fullname" . }}-service-account
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Values.rbac.roleRef }}
```

Comment:
@neumanndaniel  I am new to k8s rbac. Can you help me understand what .Values.rbac.roleRef is set to and where it is set, when rbac is enabled?

Reply:
The cluster role will be cluster-admin. https://github.com/Azure/iot-edge-virtual-kubelet-provider/pull/13/files

Currently, it is the same role setup as for the Virtual Kubelet for Azure Container Instances. It should be possible to create a custom cluster role with least privilege for the VK. But using the same approach as the VK for ACI was the fastet way right now to add RBAC support.